install_config/syncing_groups_with_ldap.adoc

```diff
@@ -229,7 +229,7 @@ $ oadm groups prune --sync-config=config.yaml --confirm
 ----
 
 [[sync-examples]]
-== Sync Examples
+== Direct Membership Sync Examples
```

Comment:
Still think changing the name of the core example section just to differentiate it more from a small example at the end of the doc is not the best.